libs/input-mapping/src/Table/Strategy/AbstractWorkspaceStrategy.php

```diff
@@ -343,30 +343,32 @@ private function buildCloneInputWithLoadType(RewrittenInputTableOptions $table):
 
     private function decideTableLoadMethod(RewrittenInputTableOptions $table, array $loadOptions): WorkspaceLoadType
     {
-        // Validate that table can be loaded to this workspace type
-        LoadTypeDecider::checkViableLoadMethod(
-            $table->getTableInfo(),
-            $this->getWorkspaceType(),
-            $loadOptions,
-            $this->clientWrapper->getToken()->getProjectId(),
-        );
+        if ($this->getWorkspaceType() === 'bigquery') {
```

Comment:
tenhle if byl schválně schovanej v tom LoadTypeDecideru, aby se to líp testovalo a byly ty podmínky na jednom místě

Reply:
> pak mám pocit, že z toho vypadla kombinace, že BQ workspace s neBQ tabulkou a obráceně může udělat jen COPY

to jsem nechal tu - https://github.com/keboola/platform-libraries/pull/455/files#diff-80555860977369aedbefec00bfd988ac37b20661a0e89dabca6ca6c571dceb2bR19
ale je to teda pro oboje, BQ workspace s neBQ tabulkou nejde ani pro jedno